samples/State-Management-Bot/config.yaml

```diff
@@ -0,0 +1,4 @@
+Settings:
```

Comment:
Your call, but it is possible to add license headers in comments in `.yaml` via the `#` character.

It's technically a config file and analogous to our appsettings.json and .env files which don't have license headers.... But the docs below are clear on when headers should be used.

([Link to official guidance](https://docs.opensource.microsoft.com/releasing/copyright-headers.html))